.eslintrc.js

```diff
@@ -0,0 +1,334 @@
+const kIndent = 2
+
+const kBaseBlankLineBetweenStatementsConfig = [
+  {
+    blankLine: 'always',
+    prev: 'block-like',
+    next: '*'
+  },
+  {
+    blankLine: 'always',
+    prev: 'break',
+    next: '*'
+  },
+  {
+    blankLine: 'always',
+    prev: ['case', 'default'],
+    next: '*'
+  },
+  {
+    blankLine: 'always',
+    prev: '*',
+    next: 'class'
+  },
+  {
+    blankLine: 'always',
+    prev: '*',
+    next: 'default'
+  },
+  {
+    blankLine: 'always',
+    prev: '*',
+    next: 'do'
+  },
+  {
+    blankLine: 'always',
+    prev: '*',
+    next: 'for'
+  },
+  {
+    blankLine: 'always',
+    prev: '*',
+    next: 'function'
+  },
+  {
+    blankLine: 'always',
+    prev: '*',
+    next: 'if'
+  },
+  {
+    blankLine: 'always',
+    prev: '*',
+    next: 'iife'
+  },
+  {
+    blankLine: 'always',
+    prev: '*',
+    next: 'switch'
+  },
+  {
+    blankLine: 'always',
+    prev: 'case',
+    next: 'case'
+  },
+  {
+    blankLine: 'always',
+    prev: '*',
+    next: 'try'
+  },
+  {
+    blankLine: 'always',
+    prev: '*',
+    next: 'while'
+  }
+]
```

Comment:
I would rather just get rid of this. It's not solving a problem I can see. I'd rather err on the side of fewer linting rules